requirements.txt

```diff
@@ -1,6 +1,9 @@
+contextdecorator==0.10.0
 freezegun==0.3.9
 mock==2.0.0
+pbr==4.0.2
+py==1.5.3
 pytest==3.2.1
 python-dateutil==2.6.1
+pytz==2018.6
```

Comment:
From what I can tell, this requirements file isn't being used by `setup.py`, right? Because if it is, the `pytz` under `install_requires` might be redundant.

Reply:
It's only used for the dev env / tests -- eg, it includes pytest, but the setup.py doesn't.